Modules/_operator.c

```diff
@@ -698,6 +698,20 @@ _operator_index(PyObject *module, PyObject *a)
     return PyNumber_Index(a);
 }
 
+/*[clinic input]
+_operator.is_none = _operator.truth
+
+Same as a is None.
+[clinic start generated code]*/
+
+static int
+_operator_is_none_impl(PyObject *module, PyObject *a)
+/*[clinic end generated code: output=b3e4655e6929eda6 input=fdfffa61fae7a975]*/
+{
+    PyObject *result = Py_IsNone(a) ? Py_True : Py_False;
+    return Py_NewRef(result);
```

Comment:
there's a lot of compiler warnings on this line:

```
returning ‘PyObject *’ {aka ‘struct _object *’} from a function with return type ‘int’ makes integer from pointer without a cast [-Wint-conversion]
```

and tests are failing

Reply:
Currently investigating this, thanks for the comment

Reply:
Stupid mistake - used `truth` instead of `neg` as operator remplate for clinic. Sorry for that!